#534 implement record relation within the pojo class (#542)
boozallen/aissemble: DRAFT_RELEASE_NOTES.md

## Spark Upgrade
Spark and PySpark have been upgraded from version 3.5.2 to 3.5.4.

## Record Relation

> To better track of nested data, we have added a new relation feature inside of record, which allows one record reference another. For more detail, refer to [Record Relation Options](https://boozallen.github.io/aissemble/aissemble/current-dev/record-metamodel.html#_record_relation_options)

Reviewer: ```suggestion
To enable nested data records, we have added a new relation feature to the record metamodel. This allows records to reference other records. For more details, refer to the [Record Relation Options].(https://boozallen.github.io/aissemble/aissemble/current-dev/record-metamodel.html#_record_relation_options)
```
